README.md

```diff
@@ -8,140 +8,130 @@
+        configureKeyboardSupport(with: configuration)
     }
 }
 ```
 
-You can also set the delegate of the KeyboardManager to get a callback when "Done" is tapped on the keyboard.
+2. Supporting a bottom constraint of a view.
+    Passing a constraint offset can be useful when dealing with tab bars.
 ``` swift
-override func viewDidLoad() {
-    super.viewDidLoad()
+import KeyboardSupport
 
-    keyboardManager = KeyboardManager(textFields: [textField1, textField2], configuresReturnKeys: true)
-    keyboardManager?.delegate = self
-}
-```
+class ViewController: KeyboardSupportViewController {
+    @IBOutlet private var bottomConstraint: NSLayoutConstraint!
 
-``` swift
-extension ViewController: KeyboardManagerDelegate {
-    func keyboardManagerDidTapDone(_ manager: KeyboardManager) {
-        // Handle "Done" tap
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let configuration = KeyboardSupportConfiguration(bottomConstraint: bottomConstraint, constraintOffset: 49, usesDismissalView: true)
+        configureKeyboardSupport(with: configuration)
     }
 }
 ```
 
-### KeyboardManager with custom view above the keyboard.
-Create a custom class that is a subclass of UIView or one of its subclasses to pass to the KeyboardManager. Your custom class can conform to KeyboardInputAccessory so the KeyboardManager handles navigation between text fields. Otherwise, your view controller can handle navigation callbacks.
+3. Supporting dismissal of the keyboard by tapping outside a text field.
+    Make sure usesDismissalView is set to true.
 ``` swift
-class KeyboardToolbar: UIToolbar, KeyboardInputAccessory {
-    // Use the delegate
-    weak var keyboardInputAccessoryDelegate: KeyboardInputAccessoryDelegate?
-
-    // Example button actions
-    @IBAction func backButtonTapped(_ sender: UIButton) {
-        keyboardInputAccessoryDelegate?.keyboardInputAccessoryDidTapBack(self)
-    }
-
-    @IBAction func nextButtonTapped(_ sender: UIButton) {
-        keyboardInputAccessoryDelegate?.keyboardInputAccessoryDidTapNext(self)
-    }
-
-    @IBAction func doneButtonTapped(_ sender: UIButton) {
-        keyboardInputAccessoryDelegate?.keyboardInputAccessoryDidTapDone(self)
-    }
-}
+var configuration = KeyboardSupportConfiguration()
+configuration.usesDismissalView = true
+configureKeyboardSupport(with: configuration)
 ```
 
-### KeyboardDismissable
-Conform to this protocol to enable keyboard dismissal via tapping the screen when the keyboard is displayed.
+4. Supporting the keyboard's "Return" key to navigate linearly from one text field to the next.
+    Make sure usesKeyboardNextButtons is set to true.
 ``` swift
-class ViewController: UIViewController, KeyboardDismissable {
+var configuration = KeyboardSupportConfiguration()
+configuration.usesKeyboardNextButtons = true
+configureKeyboardSupport(with: configuration)
+```
 
+5. Supporting your custom toolbar above the keyboard.
```

Comment:
If we can, we should include an image of what the `KeyboardToolbar` looks like in this example to better depict this feature.

Reply:
I can add a screenshot.